utils/misc.ts

```diff
@@ -0,0 +1,7 @@
+export const isInIframe = () => {
+  try {
+    return document.location.hostname !== window.parent.location.hostname;
```

Comment:
`document.location` is an object, we're most interested in hostname

Reply:
> `document.location` is an object, we're most interested in hostname

I've just checked it and it looks that comparing locations work fine with the whole object instead of `hostname` only.